### DISCUSSION TITLE
Apply style to bulk actions on the td element.

### DISCUSSION THREAD

**Discussion by saloeric** [rappasoft/laravel-livewire-tables#2109] [General]
Hello, the style for bulk actions is not being applied to the columns. 

            ->setTdAttributes(function (Column $column, $row, $columnIndex, $rowIndex) {
                if ($column->getTitle() == 'bulkactions') {
                    return [
                        'class' => 'bg-yellow-500 dark:bg-yellow-800',
                        'default' => false,
                        'default-colors' => false,
                    ];
                }
                if (in_array($columnIndex, [0])) {
                    return [
                        'default' => false,
                        'class' => 'p-4 w-[1px] text-center',
                    ];
                }
                if (in_array($columnIndex, [7, 9])) {
                    return [
                        'default' => false,
                        'class' => 'p-4 w-[1px]',
                    ];
                }
                if (in_array($columnIndex, [1, 2, 3, 6, 8])) {
                    return [
                        'default' => false,
                        'class' => 'p-4 whitespace-nowrap w-[1px]',
                    ];
                }

                return [
                    'default' => false,
                    'class' => 'p-4',
                ];
            })

Could you help? Thank you.

**Comment by saloeric** [comment]
It works with the method: setBulkActionsTdAttributes

Thank you

**Reply by lrljoe** [reply]
So it all works as documented?

**Reply by saloeric** [reply]
this is not

    if ($column->getTitle() == 'bulkactions') {
                return [
                    'class' => 'bg-yellow-500 dark:bg-yellow-800',
                    'default' => false,
                    'default-colors' => false,
                ];
            }
            
            
Thank you
